packages/engine-render/src/components/sheets/constants.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const BORDER_Z_INDEX = 50;
+export const FONT_EXTENSION_Z_INDEX = 45;
+export const BG_Z_INDEX = 21;
+export const PRINTING_BG_Z_INDEX = 21;
+
+export const EXPAND_SIZE_FOR_RENDER_OVERFLOW = 20;
```

Comment:
should we provide a function or poperty fot user to customize the overfollow testing scope

Reply:
customize in facade?

Reply:
Add it to config service.